packages/component-library/stories/HorizontalBarChart.story.js

```diff
@@ -2,122 +2,237 @@ import React from "react";
-];
-
-const sampleMinimalistData = [
-  { sortOrder: 1, population: 2000, label: "Labrador Retriever" },
-  { sortOrder: 2, population: 8000, label: "Standard Poodle" }
-];
-
-const sampleUnsortedData = [
-  { population: 2000, label: "Labrador Retriever" },
-  { population: 8000, label: "Standard Poodle" },
-  { population: 6000, label: "French Bulldog" },
-  { population: 3000, label: "Afghan Hound" },
-  { population: 1000, label: "Jack Russell Terrier" }
-];
+const GROUP_IDS = {
+  LABELS: "Labels",
+  DATA: "Data",
+  CUSTOM: "Custom"
+};
+const getKeyNames = obj => {
+  const keyNames = {};
+  Object.keys(obj).forEach(key => {
+    keyNames[key] = key;
+  });
+  return keyNames;
+};
 
 export default () =>
   storiesOf("Component Lib|Charts/Horizontal Bar Chart", module)
     .addDecorator(checkA11y)
     .addDecorator(withKnobs)
-    .add("Default sort order", () => {
-      const data = object("Data", sampleUnsortedData);
-      const dataValue = text("Data values", "population");
-      const dataLabel = text("Data series labels", "label");
+    .add(
+      "Standard",
+      () => {
+        const sampleData = [
+          { sortOrder: 1, population: 2000, label: "Labrador Retriever" },
+          { sortOrder: 2, population: 8000, label: "Standard Poodle" },
+          { sortOrder: 3, population: 6000, label: "French Bulldog" },
+          { sortOrder: 4, population: 3000, label: "Afghan Hound" },
+          { sortOrder: 5, population: 1000, label: "Jack Russell Terrier" }
+        ];
+        const title = text("Title", "Dogs and their Money", GROUP_IDS.LABELS);
+        const subtitle = text(
+          "Subtitle",
+          "As of January 2017",
+          GROUP_IDS.LABELS
+        );
+        const xLabel = text("X-axis label", "Dollars", GROUP_IDS.LABELS);
+        const dataValueFormatter = getKeyNames(civicFormat);
+        const optionSelectX = options(
+          "X-axis value format",
+          dataValueFormatter,
+          "dollars",
+          { display: "select" },
+          GROUP_IDS.LABELS
+        );
+        const yLabel = text("Y-axis label", "Dogs", GROUP_IDS.LABELS);
+        const sortOrder = text("Sort order", "sortOrder", GROUP_IDS.DATA);
+        const dataValue = text("Data value", "population", GROUP_IDS.DATA);
+        const dataLabel = text("Data label", "label", GROUP_IDS.DATA);
+        const data = object("Data", sampleData, GROUP_IDS.DATA);
 
-      return (
-        <HorizontalBarChart
-          data={data}
-          dataValue={dataValue}
-          dataLabel={dataLabel}
-        />
-      );
-    })
-    .add("No title", () => {
-      const data = object("Data", sampleData);
-      const sortOrder = text("Data series", "sortOrder");
-      const dataValue = text("Data values", "population");
-      const dataLabel = text("Data series labels", "label");
+        return (
+          <HorizontalBarChart
+            data={data}
+            sortOrder={sortOrder}
```

Comment:
Technically, sortOrder isn't required, the component will pick a sortOrder by default. We should move it to the custom story.

A follow up issue for this is that the default sort should be changed to sort from highest value to lowest value, rather than the current sorting method.